recipes/isort/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "4.2.5" %}
+{% set md5 = "71489ebd936429d5bda2af7ae662ed78" %}
+
+package:
+  name: isort
+  version: {{ version }}
+
+source:
+  fn: isort-{{version}}.tar.gz
+  url: https://pypi.python.org/packages/70/65/49f66364f4ac551ec414e88537b02be439d1d9ea7e1fdd6d526fb8796bf9/isort-{{version}}.tar.gz
```

Comment:
Change the URL to `https://pypi.io/packages/source/i/isort/isort-{{ version }}.tar.gz`

(That PyPI URL is terrible.)